src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -2641,8 +2641,18 @@ class PhysicsDescription:
         default=None, validator=optional(instance_of(Path))
     )
     keep_former_results: bool = attr.ib(default=False, validator=instance_of(bool))
-    emulsion_model = attrib_enum(default=constants.EmulsionModelType.ModelDefault)
+    emulsion_model_enabled: bool = attr.ib(default=True, validator=instance_of(bool))
```

Comment:
Update the changelog and document this breaking change.
This is a API and an `.alfacase` breaking change.

Check on prior changelog entries how those are documented plz.